src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Figure extends Area {
```

Comment:
According to the checklist, interfaces should not be used to group different behaviors. The `draw()` and `getArea()` methods are conceptually different from each other and should not be in the same interface. Consider splitting this interface into two separate ones, each with a single responsibility.